tests/test_integration.py

```diff
@@ -749,5 +768,149 @@ def test_train_single_task_regression_reaction(self,
             mean_score = test_scores.mean()
             self.assertAlmostEqual(mean_score, expected_score, delta=DELTA*expected_score)
 
+    @parameterized.expand([
+        (
+                'chemprop',
+                'chemprop',
+                'auc',
+                0.699453,
+                ['--number_of_molecules', '2', '--data_path', os.path.join(TEST_DATA_DIR, 'classification_multimolecule.csv')]
+        )
+    ])
+    def test_single_task_multimolecule_classification(self,
+                                          name: str,
+                                          model_type: str,
+                                          metric: str,
+                                          expected_score: float,
+                                          train_flags: List[str] = None):
+        with TemporaryDirectory() as save_dir:
+            # Train
+            self.train(
+                dataset_type='classification',
+                metric=metric,
+                save_dir=save_dir,
+                model_type=model_type,
+                flags=train_flags
+            )
+
+            # Check results
+            test_scores_data = pd.read_csv(os.path.join(save_dir, TEST_SCORES_FILE_NAME))
+            test_scores = test_scores_data[f'Mean {metric}']
+
+            mean_score = test_scores.mean()
+            self.assertAlmostEqual(mean_score, expected_score, delta=DELTA * expected_score)
+
+    @parameterized.expand([
+        (
+                'chemprop',
+                'chemprop',
+                ['--fingerprint_type', 'MPN'],
+        ),
+        (
+                'chemprop',
+                'chemprop',
+                ['--fingerprint_type', 'last_FFN']
+        )
+    ])
+    def test_single_task_fingerprint(self,
+                                            name: str,
+                                            model_type: str,
+                                            fingerprint_flags: List[str],
+                                            train_flags: List[str] = None,
+                                     ):
+        with TemporaryDirectory() as save_dir:
+            # Train
+            dataset_type = 'classification'
+            self.train(
+                dataset_type=dataset_type,
+                metric='auc',
+                save_dir=save_dir,
+                model_type=model_type,
+                flags=train_flags
+            )
+
+            # Predict
+            fingerprint_path = os.path.join(save_dir, 'fingerprints.csv')
+            self.fingerprint(
+                dataset_type=dataset_type,
+                checkpoint_dir=save_dir,
+                fingerprint_path=fingerprint_path,
+                fingerprint_flags=fingerprint_flags
+            )
+
+            # Check to make sure that fingerprints are generated for all input molecules
```

Comment:
This is a good test. It's more of a unit test than an integration test. But since a different location for unit tests is in progress (#232) and not yet in place, we can have it here for now.